packages/router/__tests__/router-test.ts

```diff
@@ -4353,7 +4466,7 @@ describe("a router", () => {
       await N.loaders.root.resolve("ROOT_DATA*");
       await N.loaders.tasks.resolve("TASKS_DATA");
       expect(t.router.state).toMatchObject({
-        historyAction: "PUSH",
+        historyAction: "REPLACE",
```

Comment:
Because the router was expecting `useSubmitImpl` to pass the right history action before, we had some prior tests that were expecting POST navigations to be a PUSH